src/benchmarklib/synthetic_table_generator.hpp

```diff
@@ -32,7 +32,7 @@ struct ColumnDataDistribution {
 
   static ColumnDataDistribution make_skewed_normal_config(const double skew_location = 0.0,
                                                           const double skew_scale = 1.0,
-                                                          const double skew_shape = 0.0) {
+                                                          const double skew_shape = 0.0001) {
```

Comment:
still necessary?

Reply:
Not with the latest fixes to boost, but as long as we do not pull the latest master of boost, we have use this fix.

Reply:
I had a few things not pushed yet, sorry for that. There is a comment with a link to the issue.